app/assets/stylesheets/components/_option-select.scss

```diff
@@ -110,6 +110,7 @@
 
   .govuk-form-group {
     padding-top: govuk-spacing(2);
+    margin-bottom: $gutter-one-third;
```

Comment:
Should we be using `govuk-spacing(2)` here?

Reply:
Probably, but as this is one of three uses of gutter in this file, I think we should look at it later.